<?xml version="1.0"?>
<launch>
       <arg name="navdata_topic_name" 	default="/navdata_uav" />
       <arg name="input_cmd_topic_name" 	default="/bebop/cmd_input" />
       <arg name="output_cmd_topic_name" 	default="/bebop/cmd_vel" />
    
	<!-- Launches the generate geo cmd node -->
    <node pkg="multi_drone_safe_ctrl" type="gen_geo_cmd_node" name="gen_geo_cmd_node" output="screen">

        <param name="debug" value="true" />
        <param name="Vmax" value="0.4" />
        <param name="l" value="0.5" />
        <param name="rm" value="0.8" />
        <param name="ra" value="2.0" />
        <param name="rh" value="3.0" />
        <param name="rs" value="0.01" />
        <param name="e" value="0.01" />
        <param name="kx" value="1.0" />
        <param name="ky" value="1.0" />
        <param name="x_b" value="6.0" />
        <param name="y_b" value="6.0" />
        <param name="gain_cmd" value="0.5"/>
        <param name="Hz" value="20.0"/>

        <remap from="/cmd_input" to="$(arg input_cmd_topic_name)"/>    
        <remap from="/navdata_uav" to="$(arg navdata_topic_name)"/>    
        <remap from="/uav_cmd_vel" to="$(arg output_cmd_topic_name)"/> 
    </node>
</launch>
